blathering-barnacles/Bangazon-2
# User can view and edit their account settings (ticket #18)

Description:
Added the ability to view and edit user settings.  Steps completed: Added new URLPattern that takes the user to the userSettings view when the user clicks the User Settings button on the navbar.  User Settings display First Name, Last name, phone, address, all payment options listed and all purchase history.  User can click the Edit Settings button which takes them to a page where they can edit last name, address and phone number.  Saving these settings takes the user back to the user settings page.

Steps to Test:

go to localhost:8000
Log into Bangazon
Click the "User Settings" button on the navbar
View the user settings displayed to the screen 
Click the Edit Settings button and go to Edit Settings page
Make changes and verify that the changes save on the User Settings page

Link to Feature Ticket:

#18 [https://github.com/blathering-barnacles/Bangazon-2/issues/18](url)
